teamworkApp/lib/importData.py

```diff
@@ -165,13 +165,13 @@ def list_answer_data(student_dict):
 		question_count       = 0
 
 		for response in enummed_responses:
-			answer_to_db += [
+			answer_to_db.append(
 			(response,
 			student_dict[student][0],
 			now,
 			primary_keys[count],
 			question_count)
-			]
+			)
```

Comment:
My only comment and this is super nit picky is that I think you should indent the inner tuple another line here (and the other places like that). I would do 
```
answer_to_db.append((
    response,
    student_dict[student][0],
    now,
    primary_keys[count],
    question_count,
))
```
Although its your code so if you don't agree, I'm not going to complain. (maybe also do the parens on separate lines, but then there is a lot of indenting)